spec/api/v1/dropdown_data_spec.rb

```diff
@@ -81,6 +81,7 @@
 
     it "should return a JSON formatted list of the required information" do
       results.each do |endpoint, json|
+        ap "#{endpoint}"
```

Comment:
Is this due to a flickering test?

Reply:
oops, good catch - debug statement left in 👍